sw/tools/sbus_fakerator/sbus_fakerator

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+python sw/tools/sbus_fakerator/main.py $1 $2
```

Comment:
I dont think I follow - can you elaborate? This works just fine so why not use it as is?